### PR TITLE
Implement list menu with price and categories

### PR DESCRIPTION
**Pull request by caressahh**
User stories:
#1 
#2

### CODE REVIEW

File: features/step_definitions/base_steps.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+Given(/^I visit the menu page$/) do
+  visit '/'
+end
+
+Given(/^the following dishes exist$/) do |table|
+  table.hashes.each do |dish|
+    dish[:price]=dish[:price].to_i
```
Comment (andycarnahan) [inline features/step_definitions/base_steps.rb]: Put spaces around the '='.

File: features/step_definitions/base_steps.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+Then(/^I should see "([^"]*)"$/) do |content|
+  expect(page).to have_content content
+end
+
+Then(/^show me the page$/) do
+  save_and_open_page
+end
+
+Then(/^dish prices$/) do
```
Comment (andycarnahan) [inline features/step_definitions/base_steps.rb]: Delete this step definition.

File: features/step_definitions/base_steps.rb [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  pending # Write code here that turns the phrase above into concrete actions
+end
+
+Then(/^show med the page$/) do
```
Comment (andycarnahan) [inline features/step_definitions/base_steps.rb]: I think this should just be removed.

**Review by andycarnahan — CHANGES_REQUESTED** [review]
Good job! Just a couple small fixes.

File: features/step_definitions/base_steps.rb (hunk 1 above)
Comment (caressahh) [reply features/step_definitions/base_steps.rb]: Yes, thanks!

File: features/step_definitions/base_steps.rb (hunk 2 above)
Comment (caressahh) [reply features/step_definitions/base_steps.rb]: Great catch! fixed!

File: features/step_definitions/base_steps.rb (hunk 3 above)
Comment (caressahh) [reply features/step_definitions/base_steps.rb]: GREAT STUFF!

**Review by andycarnahan — APPROVED** [review]

**Pull request merged**